<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharacterWidget</class>
 <widget class="QWidget" name="CharacterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1303</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="vLayoutCharacter">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabProfileEquip">
      <attribute name="title">
       <string>프로필 / 장비</string>
      </attribute>
      <layout class="QVBoxLayout" name="vLayoutTabProfileEquip">
       <property name="spacing">
        <number>1</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="hLayoutProfile">
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupName">
           <property name="minimumSize">
            <size>
             <width>168</width>
             <height>125</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>168</width>
             <height>125</height>
            </size>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QVBoxLayout" name="vLayoutName">
            <property name="spacing">
             <number>1</number>
            </property>
            <property name="leftMargin">
             <number>9</number>
            </property>
            <item>
             <widget class="QPushButton" name="pbSibling">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>보유 캐릭터</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lbName">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>-</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupTitleGuild">
           <property name="minimumSize">
            <size>
             <width>168</width>
             <height>125</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>168</width>
             <height>125</height>
            </size>
           </property>
           <property name="title">
            <string>칭호 / 길드</string>
           </property>
           <layout class="QVBoxLayout" name="vLayoutTitleGuild">
            <property name="spacing">
             <number>1</number>
            </property>
            <item>
             <widget class="QLabel" name="lbTitle">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>-</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lbGuild">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>-</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupServerClass">
           <property name="minimumSize">
            <size>
             <width>143</width>
             <height>125</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>143</width>
             <height>125</height>
            </size>
           </property>
           <property name="title">
            <string>서버 / 직업</string>
           </property>
           <layout class="QVBoxLayout" name="vLayoutServerClass">
            <property name="spacing">
             <number>1</number>
            </property>
            <item>
             <widget class="QLabel" name="lbServer">
              <property name="minimumSize">
               <size>
                <width>125</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>125</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>-</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lbClass">
              <property name="minimumSize">
               <size>
                <width>125</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>125</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>-</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupLevel">
           <property name="minimumSize">
            <size>
             <width>143</width>
             <height>125</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>143</width>
             <height>125</height>
            </size>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QVBoxLayout" name="vLayoutLevel">
            <property name="spacing">
             <number>1</number>
            </property>
            <item>
             <widget class="QLabel" name="lbBattleLevel">
              <property name="minimumSize">
               <size>
                <width>125</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>125</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>-</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lbExpLevel">
              <property name="minimumSize">
               <size>
                <width>125</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>125</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>-</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lbItemLevel">
              <property name="minimumSize">
               <size>
                <width>125</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>125</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>-</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupAbility">
           <property name="minimumSize">
            <size>
             <width>227</width>
             <height>125</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>227</width>
             <height>125</height>
            </size>
           </property>
           <property name="title">
            <string>특성</string>
           </property>
           <layout class="QHBoxLayout" name="hLayoutAbility">
            <item>
             <layout class="QVBoxLayout" name="vLayoutAbility1">
              <property name="spacing">
               <number>1</number>
              </property>
              <item>
               <widget class="QLabel" name="lbCritical">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>치명</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lbSpecification">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>특화</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lbSwiftness">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>신속</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="vLayoutAbility2">
              <property name="spacing">
               <number>1</number>
              </property>
              <item>
               <widget class="QLabel" name="lbDomination">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>제압</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lbEndurance">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>인내</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lbExpertise">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>숙련</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupCard">
           <property name="minimumSize">
            <size>
             <width>318</width>
             <height>125</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>318</width>
             <height>125</height>
            </size>
           </property>
           <property name="title">
            <string>카드</string>
           </property>
           <layout class="QHBoxLayout" name="hLayoutCard">
            <item>
             <layout class="QVBoxLayout" name="vLayoutCard"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupCollectible">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>125</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>550</width>
             <height>125</height>
            </size>
           </property>
           <property name="title">
            <string>수집품</string>
           </property>
           <layout class="QGridLayout" name="gridGroupCollectible"/>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hLayoutEquip">
         <property name="spacing">
          <number>1</number>
         </property>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hLayoutAccessory">
         <property name="spacing">
          <number>1</number>
         </property>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hLayoutStoneBraceletEngrave">
         <property name="spacing">
          <number>1</number>
         </property>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hLayoutGem1">
         <property name="spacing">
          <number>1</number>
         </property>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hLayoutGem2">
         <property name="spacing">
          <number>1</number>
         </property>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hLayoutGem3">
         <property name="spacing">
          <number>1</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSkill">
      <attribute name="title">
       <string>스킬</string>
      </attribute>
      <layout class="QGridLayout" name="gridTabSkill"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
